phone/app/src/main/java/jp/ac/mayoi/maigocompass/NavHost.kt

```diff
@@ -14,19 +20,27 @@ import jp.ac.mayoi.core.navigation.TravelingNavigation
 @Composable
 fun PhoneNavHost(
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/9148201e5a65d73d1eddb227e23c2669828ffbe0/phone/app/src/main/java/jp/ac/mayoi/maigocompass/NavHost.kt#L21"></span>Parameters in a composable function should be ordered following this pattern: params without defaults, modifiers, params with defaults and optionally, a trailing function that might not have a default param.<br />
Current params are: [navController: NavHostController = rememberNavController(), innerPadding:PaddingValues] but should be [innerPadding:PaddingValues, navController: NavHostController = rememberNavController()].<br />
See https://slackhq.github.io/compose-lints/rules/#ordering-composable-parameters-properly for more information.</td>
    </tr>
  </tbody>
</table>